activemq-client/src/main/java/org/apache/activemq/management/UnsampledStatisticImpl.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.management;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * UnsampledStatistic<T> implementation
+ * 
+ */
+public class UnsampledStatisticImpl<T> extends StatisticImpl implements UnsampledStatistic<T> {
+
+    protected final AtomicReference<T> value = new AtomicReference<>();
```

Comment:
Is the intent to have another class subclass this at some point? I'm wondering why these are protected and not private.

Reply:
Good catch. I’ll tidy that up